reference/orders.v2.oas2.yml

```diff
@@ -4746,3 +4746,492 @@ components:
+          description: Text code identifying the BigCommerce shipping module selected by the customer.
+          example: Free Shipping
+    shippingConsignment_Get:
+      description: ''
+      allOf:
+        - type: object
+          properties:
+            id:
+              type: integer
+              example: 99
+              description: This is the same ID as that of `/stores/{store_hash}/v2/orders/{order_id}/shipping_addresses`
+        - $ref: '#/components/schemas/shippingConsignment_Base'
+        - type: object
+          properties:
+            line_items:
+              type: array
+              items:
+                $ref: '#/components/schemas/products_Resource'
+            items_total:
+              type: number
+              example: 1
+              description: The total number of items in the order.
+            items_shipped:
+              type: number
+              description: The number of items that have been shipped.
+              example: 0
+            base_cost:
+              type: number
+              description: The base shipping cost value.
+              example: 5.1
+            cost_ex_tax:
+              type: number
+              example: 5.1
+              description: The shipping cost value excluding tax.
+            cost_inc_tax:
+              type: number
+              description: The shipping cost value including tax.
+              example: 6.2
+            cost_tax:
+              type: number
+              description: The tax amount on the shipping cost.
+              example: 1.1
+            cost_tax_class_id:
+              type: integer
+              description: 'The ID of the tax class applied to the shipping cost. (NOTE: Value is ignored if automatic tax is enabled.)'
+              example: 2
+            base_handling_cost:
+              type: number
+              description: The base handling charge.
+              example: 4.1
+            handling_cost_ex_tax:
+              type: number
+              description: The handling charge, excluding tax.
+              example: 4.1
+            handling_cost_inc_tax:
+              type: number
+              description: The handling charge, including tax.
+              example: 5.3
+            handling_cost_tax:
+              type: number
+              example: 1.2
+            handling_cost_tax_class_id:
+              type: integer
+              description: 'The ID of the tax class applied to the handling charge. (NOTE: Value is ignored if automatic tax is enabled.)'
+              example: 2
+            shipping_zone_id:
+              type: number
+              description: Numeric ID of the shipping zone.
+              example: 1
+            shipping_zone_name:
+              type: string
+              description: Name of the shipping zone.
+              example: United States
+            shipping_quotes:
+              $ref: '#/components/schemas/shippingQuotesConsignment_Resource'
```

Comment:
Hey @bigcommerce/orders it looks like we're calling for a schema that doesn't exist in the PR. Do we have that schema available? CC: @donald-nguyen-bc @WillemHoman